File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--updating-a-chart-in-a-document-using-radio-buttons_4.vb
    Private Sub columnChart_CheckedChanged(ByVal sender As System.Object, ByVal e As System.EventArgs) _
        Handles columnChart.CheckedChanged

        If (CType(sender, RadioButton).Checked) Then

            Me.selectedType = Microsoft.Office.Interop.Graph.XlChartType.xlColumnClustered
            RaiseEvent SelectionChanged(Me, EventArgs.Empty)
        End If
    End Sub

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--updating-a-chart-in-a-document-using-radio-buttons_4.cs
        private void columnChart_CheckedChanged(object sender, EventArgs e)
        {
            if (((RadioButton)sender).Checked)
            {
                this.selectedType = Microsoft.Office.Interop.Graph.XlChartType.xlColumnClustered;
                if (this.SelectionChanged != null)
                {
                    this.SelectionChanged(this, EventArgs.Empty);
                }
            }
        }